api/handler.py

```diff
@@ -36,16 +48,45 @@ def postLocation(event, context):
 
 
 def postInfected(event, context):
+    # get db connection
+    print(f"processing infected {event}")
+    conn = make_conn()
+    cursor = conn.cursor()
+    # set the user to infected
+    cursor.execute(
+        f"update contact_tracer.device set infected = true, infected_date = '{event['infectionDate']}' where device_id = '{event['deviceId']}'; ")
+    # get all devices that need to be notified
+    cursor.execute(f"""
+    select distinct dl3.device_id as notifyDeviceId
+    from 
+    contact_tracer.device_location dl 
+    join lateral 
+    (
+        select * from contact_tracer.device_location where device_id != dl.device_id and ST_DistanceSphere(location,dl.location) < 500
```

Comment:
Maybe we should use the second parameter of `execute` so we can claim that we're robust against sql injection? But perhaps do this in a second iteration and work on more functionality.

Reply:
good point